src/Stache/Stores/BasicStore.php

```diff
@@ -52,11 +55,13 @@ protected function cacheItem($item)
 
         $cacheKey = $this->getItemCacheKey($key);
 
+        Blink::forget($cacheKey);
```

Comment:
As per the suggestion in the comment, it seems like a good idea to just put it back into the Blink cache, rather than forgetting it.

`Blink::put($cacheKey, $item);`

Reply:
Hey, I agree this way would be better, and I had tried to do this initially - my issue was it was causing all the tests in `tests/Feature/Entries/LocalizeEntryTest.php` to fail when it was set to put. I spent a while debugging it, but couldn't work out what was going on. I'm guessing when `fresh()` is called on the entry it clears the cache, but I couldn't find where it was doing that clear to put the blink clear with it - and was also worried there'd be similar issues elsewhere. Not sure if you, or anyone has any ideas of what could be happening?